extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1361,6 +1362,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
 
     this.switchTo_(pageId, NavigationDirection.PREVIOUS);
+    this.storeService_.dispatch(Action.TOGGLE_PAUSED, false);
```

Comment:
Nit: here and below, can you invert these two lines? Unpausing the story before the switchTo will run a bit less code and go through the usual (and better tested) code path. Not a big deal